src/components/Card/src/stories/Paper.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { Meta, Story } from '@storybook/react';
-import Typography from '@gemeente-denhaag/typography';
+import { Typography } from '@material-ui/core';
```

Comment:
I would not use the Material UI Typography component, we want our component library to use our own as much as possible.

Reply:
The Typography component as used in Card/Paper does not exist anymore, but it still relies on it.
Once we design the paper component this can be removed.